easybuild/easyconfigs/i/IntelPython/IntelPython-2.7.15-2019.2.066.eb

```diff
@@ -0,0 +1,43 @@
+# Authors:: Jack Perdue <j-perdue@tamu.edu> - TAMU HPRC - http://hprc.tamu.edu
+
+easyblock = 'ConfigureMake'
+
+name = 'IntelPython'
+version = '2.7.15'
+pysver = version[0]
+relver = '2019.2.066'
+versionsuffix = '-%s' % relver
+
+homepage = 'https://software.intel.com/en-us/intel-distribution-for-python'
+
+description = """
+ Intel® Distribution for Python. Powered by Anaconda.
+ Accelerating Python* performance on modern architectures from Intel.
+"""
+
+# l_pythoni2_p_2019.2.066.tar.gz from the Intel site
```

Comment:
@JackPerdue We can actually include the `source_urls` imho, since Intel allows you to simply `curl` them without authorization (and I was told they're not planning to change this).

We already do this for recent `icc`, `ifort`, `impi`, `imkl` versions too (and so do other projects).